### PR TITLE
Accordion updates

### PR DESCRIPTION
**Pull request by darintrimillos**
Got design approval from Fes, so we should be ready to go here.

### CODE REVIEW

File: src/components/accordion/__tests__/CdrAccordion.spec.js [hunk 1]
```diff
@@ -1,9 +1,109 @@
...
+    //     },
+    //     slots: {
+    //       default: 'this is some default slot text. it should take up some space'
+    //     }
+    //   });
+
+    //   expect(wrapper.vm.maxHeight).toBe(0);
+
+    //   // this is throwing a console error, but shouldn't be. Covered in this thread
+    //   // https://github.com/vuejs/vue-test-utils/issues/631
```
Comment (cowills) [inline src/components/accordion/__tests__/CdrAccordion.spec.js]: this issue looks like it was resolved already? the PR they reference as the solution was merged

Comment (cowills) [reply src/components/accordion/__tests__/CdrAccordion.spec.js]: also shouldn't this be `setProps` since opened isn't in `data()`?

**Comment by lancetharper** [comment]
Looks like `chromedriver` is outdated with the recent Chrome update

File: src/components/accordion/CdrAccordion.vue [hunk 2]
```diff
@@ -1,56 +1,139 @@
...
+
+      if (this.borderAligned) {
+        styles.push(this.modifyClassName(this.baseClass, 'border-aligned'));
+      }
+
+      return styles.join(' ');
+    },
+    focusedClass() {
+      return this.focused ? this.modifyClassName(this.baseClass, 'focused') : null;
+    },
```
Comment (KrisKnabel) [inline src/components/accordion/CdrAccordion.vue:126]: This `focused` classname logic at first glance seems unnecessary as you could use the `:focus` pseudo-class in the stylesheet, but I looked in the .scss file and it looks as if you have something going on with changing the z-index.  Is this old logic that could possibly be removed?  If not, maybe add a comment as to why the z-index change.

Comment (darintrimillos) [reply src/components/accordion/CdrAccordion.vue:126]: This is to fix a Chrome-only visual issue. I added a comment into the stylesheet to explain why it's there.

File: src/components/accordion/__tests__/CdrAccordion.spec.js (hunk 1 above)
Comment (darintrimillos) [reply src/components/accordion/__tests__/CdrAccordion.spec.js]: Yes, it should have been `setProps`. We actually don't need this anymore as it was testing some stuff that was refactored out.

Comment (darintrimillos) [reply src/components/accordion/__tests__/CdrAccordion.spec.js]: I added an additional assertion and a comment.

**Comment by cowills** [comment]
@darintrimillos before moving onto other work, can you also update the fall release notes in cedar-docs to document the breaking changes here?

**Comment by darintrimillos** [comment]
> @darintrimillos before moving onto other work, can you also update the fall release notes in cedar-docs to document the breaking changes here?

Yeah, I'll do that.

**Review by cowills — APPROVED** [review]
lgtm, animations are much snappier now 👍

**Pull request merged**